components/server/src/ome/services/RenderingBean.java

```diff
@@ -774,6 +776,90 @@ public long saveAsNewSettings() {
 
     /**
      * Implemented as specified by the {@link RenderingEngine} interface.
+     *
+     * @see RenderingEngine#updateSettings(RenderingDef)
+     */
+    @RolesAllowed("user")
+    public void updateSettings(RenderingDef settings) {
+        if (settings == null) {
+            return;
+        }
+
+        // Emulate setModel(RenderingModel)
+        RenderingModel model = settings.getModel();
+        if (model != null) {
+            setModel(model);
+        }
+        // Emulate setDefaultZ(int)
+        Integer defaultZ = settings.getDefaultZ();
+        if (defaultZ != null) {
+            setDefaultZ(settings.getDefaultZ());
+        }
+        // Emulate setDefaultT(int)
+        Integer defaultT = settings.getDefaultT();
+        if (defaultT != null) {
+            setDefaultT(settings.getDefaultT());
+        }
+        QuantumDef quantumDef = settings.getQuantization();
+        if (quantumDef != null) {
+            // Emulate setQuantumStrategy(int)
+            Integer bitResolution = quantumDef.getBitResolution();
+            if (bitResolution != null) {
+                setQuantumStrategy(bitResolution);
+            }
+            // Emulate setCodmainInterval(int, int)
+            Integer start = quantumDef.getCdStart();
+            Integer end = quantumDef.getCdEnd();
+            if (start != null && end != null) {
```

Comment:
I didn't have a concrete use case for this so didn't do it. It's not really possible now and I don't know under what conditions you would pass a `QuantumDef` instance without these filled out. Any ideas on what the behaviour should be? Should we really allow just setting the bit resolution or a `QuantumDef` with no fields populated?